access.tex

```diff
@@ -1,38 +1,64 @@
 The Rubin data rights policy is described in \citeds{RDO-013}.
+Prior to the start of Operations, all services for data access are under active development and are provided on a shared-risk basis. 
 
 \subsection{Data Access Centers}
 
-Rubin data products will be served to the community from the US Data Access Center (US DAC) hosted on the Google Cloud Platform\footnote{\url{data.lsst.cloud}}.
-\hyperref[sec:dp01]{DP0.1} and \hyperref[sec:dp02]{DP0.2}  are already available via the RSP running at the US DAC.
-
-An number of Rubin Independent Data Access Centers (IDACs) is also under construction to provide additional user computing resources to LSST users around the globe (\citeds{RTN-003}).
+Rubin data products will be served to the community from the US Data Access Center (US DAC) hosted in the Google Cloud.\footnote{\url{data.lsst.cloud}}.
+A number of Rubin Independent Data Access Centers (IDAC) are also under construction to provide additional user computing resources to LSST users around the globe (\citeds{RTN-003}).
 
 \subsection{Rubin Science Platform}
 
-The Rubin Science Platform (RSP), described in \citeds{LSE-319}, is a set of integrated web-based applications, services, and tools to query, visualize, subset, and analyze LSST data.
-It comprises three different ``Aspects'': a \emph{Portal} Aspect designed to provide an environment for data discovery, query, filtering, and visualization, a \emph{Notebook} Aspect to enable next-to-the-data analysis, and an \emph{API} Aspect that enables programmatic access to the Rubin data products via Virtual Observatory (VO) interfaces.
+The Rubin Science Platform (RSP), described in \citeds{LSE-319}, is a set of integrated web-based applications, services, and tools that provides access to the Rubin data products and enables next-to-the-data analysis. 
+The RSP comprises three different ``Aspects'': a \emph{Portal} Aspect designed to provide an environment for data discovery, query, filtering, and visualization, a \emph{Notebook} Aspect to enable next-to-the-data analysis, and an \emph{API} Aspect for programmatic access to the Rubin data products via Virtual Observatory (VO) interfaces.
 The Portal and Notebook Aspects of the RSP make use of the same APIs as the API Aspect to internally access the LSST datasets.
 
-An early version of the RSP is currently running at the US Data Access Center (US DAC) that provides access to Data Preview 0. 
+The RSP is currently under active development and a fully functional RSP is not expected to be available until DR1. 
+An early preview of the RSP is deployed at the US DAC and will be used to host the Early Science datasets.
+New functionality will be deployed incrementally, as it becomes available. 
 
-\subsubsection{Image services}
+The following functionality is already deployed and operational on the RSP:
+\begin{itemize}
+\item TAP and Butler access to catalogs and images;
```

Comment:
@gpdf do we want to mention the limited write support for butler given that this is worse than what people had with dp0.2?

Reply:
Yes - good catch. I will add it in

Reply:
My understanding is that there is no writeable Butler for DP1?

Reply:
See discussion in https://dmtn-303.lsst.io/v/DM-48071/index.html where we are talking about local sqlite for writes and some tooling to help with that. (although that tech note hasn't distinguished DP1 facilities from DR1).

Reply:
I added a comment on the Butler being read only for DP1 -- but it is not clear to me what we are advising users to do beyond save their analysis data products to their local space.

Reply:
We would hope to have some tooling to make that saving to local space a bit easier than them having to know how to save the python object to disk. The tech note talks about tooling to simplify setting up a local sqlite butler that they can `butler.put` to but it has still not been approved so the actual scope of the work is as yet undefined. cc/ @TallJimbo

Reply:
I don't want to mention anything in RTN-011 unless we are confident we will provide it. I'll add a comment that more details will come closer to the release date.

Reply:
After thinking about this  - I would prefer not to say anything so specific/technical about any part of the system in this document.  RTN-011 is a high-level strategy for enabling early science aimed at the community to help them prepare and support proposals. Such technical details are better in the online technical documentation, e.g. dp1.lsst.io

Reply:
I agree with this strategy; this isn't the place to go into specifics on something that's actually fairly difficult to explain.

What we can say at this level is just that users will be able to access and run the pipelines code in their notebooks.